src/components/CardList/CardList.jsx

```diff
@@ -0,0 +1,29 @@
+import { Flex, Grid } from '@chakra-ui/layout';
+import React, { useEffect, useState } from 'react'
+import Card from '../Card/Card'
+
+const CardList = () => {
+    const [data, setData] = useState();
```

Comment:
Most of the components don't use semicolon, but I think I've seen more lines like this one, once you decide a style I think it a good idea to keep things consistent
```suggestion
    const [data, setData] = useState()
```